packages/compose/src/hooks/use-copy-to-clipboard/index.js

```diff
@@ -27,11 +27,12 @@ function useUpdatedRef( value ) {
 /**
  * Copies the given text to the clipboard when the element is clicked.
  *
+ * @template {HTMLElement} TElementType
  * @param {string | (() => string)} text      The text to copy. Use a function if not
  *                                            already available and expensive to compute.
  * @param {Function}                onSuccess Called when to text is copied.
  *
- * @return {import('react').Ref<HTMLElement>} A ref to assign to the target element.
+ * @return {import('react').Ref<TElementType>} A ref to assign to the target element.
```

Comment:
Why was this change necessary?

Reply:
Because `Ref<HTMLElement>` isn't, assignable to `Ref<HTMLDivElement>` (but the other way around is allowed)